quadrat/block-template-parts/header.html

```diff
@@ -1,11 +1,15 @@
-<!-- wp:group {"className":"site-header","style":{"spacing":{"padding":{"right":"35px","left":"35px"}}}} -->
-	<div class="wp-block-group site-header" style="padding-right:35px;padding-left:35px">
-		<!-- wp:site-title /-->
-		<!-- wp:navigation {"orientation":"horizontal","textColor":"foreground-light","itemsJustification":"right","fontSize":"small"} -->
-			<!-- wp:navigation-link {"label":"Home","url":"#"} /-->
-			<!-- wp:navigation-link {"label":"Services","url":"#"} /-->
-			<!-- wp:navigation-link {"label":"Portfolio","url":"#"} /-->
-			<!-- wp:navigation-link {"label":"Contact","url":"#"} /-->
-		<!-- /wp:navigation -->
-	</div>
-<!-- /wp:group -->
+<!-- wp:group {"tagName":"header","style":{"spacing":{"padding":{"right":"35px","left":"35px"}}},"className":"site-header"} -->
+<header class="wp-block-group site-header" style="padding-right:35px;padding-left:35px"><!-- wp:group -->
+<div class="wp-block-group"><!-- wp:site-title /-->
+
+<!-- wp:navigation {"orientation":"horizontal","textColor":"foreground-light","itemsJustification":"right","fontSize":"small"} -->
+<!-- wp:navigation-link {"label":"Home","url":"#"} /-->
```

Comment:
I don't think we should ship navigation blocks with anything in them.  Empty blocks have better prompts for the user in the Block Editor (though the downside is the user won't see ANY navigation in the view until they add something themselves.

I'm relatively impartial but saw discussion about the same RE: TT1-Blocks